content/en/docs/concepts/overview/components.md

```diff
@@ -125,7 +125,11 @@ saving container logs to a central log store with search/browsing interface.
 
 ## {{% heading "whatsnext" %}}
 
-* Learn about [Nodes](/docs/concepts/architecture/nodes/)
-* Learn about [Controllers](/docs/concepts/architecture/controller/)
-* Learn about [kube-scheduler](/docs/concepts/scheduling-eviction/kube-scheduler/)
-* Read etcd's official [documentation](https://etcd.io/docs/)
+Learn more about the following:
+   * [Nodes](/docs/concepts/architecture/nodes/) and communication between [Nodes and control plane](/docs/concepts/architecture/control-plane-node-communication/).
+   * [Controllers](/docs/concepts/architecture/controller/) in Kubernetes.
+   * [kube-scheduler](/docs/concepts/scheduling-eviction/kube-scheduler/) is the default scheduler for Kubernetes and runs as part of the control plane.
+   * Read etcd's official [documentation](https://etcd.io/docs/).
+   * Several common [container runtimes](/docs/setup/production-environment/container-runtimes/) in Kubernetes.
+   * [cloud-controller-manager](/docs/concepts/architecture/cloud-controller/) in detail.
+   * [component tools](/docs/reference/command-line-tools-reference/) such as kubelet, kube-apiserver, kube-controller-manager, kube-proxy etc.
```

Comment:
"component tools" is a confusing thing to call these, yes they have CLIs but generally users only need to know kubectl commands

Reply:
@lavalamp,
Thanks for your review comment.
> "component tools" is a confusing thing to call these, yes they have CLIs but generally users only need to know kubectl commands

In that case, I will remove reference for component tools and add reference link for `kubectl commands`. Is that ok?
WDYT?

Reply:
Hi @lavalamp,
Could you please confirm on above.